Controller/PostController.php

```diff
@@ -23,9 +23,11 @@
 class PostController extends Controller
 {
     /**
+     * @param Request $request
+     *
      * @return RedirectResponse
      */
-    public function homeAction()
+    public function homeAction(Request $request = null)
```

Comment:
I don't think so. If you override the method, you can access the request in any case.

This parameter is always added AFAIK.